extensions/postgres/tests/postgres_test.go

```diff
@@ -22,45 +22,19 @@
 package tests
 
 import (
-	"fmt"
```

Comment:
@zklgame @duoertai This should have been done this earlier so that it's easier to add more persistence test. I am going to make another PR to do this separately, so that you can add test based on this pattern